lib/kb_cobrapy/kb_cobrapyImpl.py.bak-2019-07-25-22-19-11

```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf-8 -*-
```

Comment:
add lib/kb_cobrapy/kb_cobrapyImpl.py.bak* to `.gitignore`

Reply:
updated